app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.db_connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, second_name: str) -> None:
+        self.db_connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, second_name)"
+            f" VALUES (?, ?)",
+            (first_name, second_name)
+        )
+
+        self.db_connection.commit()
+
+    def all(self) -> list:
+        list_of_actors = self.db_connection.execute(f"SELECT * FROM"
+                                                    f" {self.table_name}")
+        return [Actor(*i) for i in list_of_actors]
+
+    def update(self,
+               new_first_name: str,
+               new_last_name: str,
+               id_to_update: int) -> None:
+        self.db_connection.execute(
+            f"UPDATE {self.table_name}"
+            " SET first_name = ?, last_name = ? WHERE id = ?",
```

Comment:
There is a mismatch in the column name `last_name` used in the SQL query. It should be `second_name` to match the database schema as defined in the `create` method.